bin/juxtaposer/tester/db/mssql/mssql.go

```diff
@@ -0,0 +1,107 @@
+package mssql
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"net/url"
+
+	_ "github.com/denisenkom/go-mssqldb"
+
+	"github.com/cyberark/secretless-broker/bin/juxtaposer/tester/api"
+	sql_db_tester "github.com/cyberark/secretless-broker/bin/juxtaposer/tester/db/sql"
+)
+
+type MssqlTester struct {
+	sql_db_tester.SqlDatabaseTester
+}
+
+func NewTester() (api.DbTester, error) {
+	tester := &MssqlTester{}
+
+	return tester, nil
+}
+
+// sqlserver://username:password@host:port?database=master&param2=value
+
+func (tester *MssqlTester) GetQueryMarkers(length int) string {
+	markerString := ""
+	for markerIndex := 1; markerIndex <= length; markerIndex++ {
+		markerString = markerString + fmt.Sprintf("@p%d, ", markerIndex)
+	}
+
+	return markerString[:len(markerString)-2]
+}
+
+func (tester *MssqlTester) Connect(options api.DbTesterOptions) error {
+	connectionOptions := map[string]string{}
```

Comment:
The `connectionOptions` map looks like it's only used inside this file, and it holds entries that parallel the entries in the `options` structure. Could you eliminate the `connectionOptions` map and just use the `options` fields directly (i.e. options.DatabaseName, options.Port, options.Host, options.Username, and options.Password)?